pages/wi/partner-listing.js

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useQuery } from '@apollo/react-hooks';
+import { gql } from 'apollo-boost';
+import { Grid, Cell } from 'styled-css-grid';
+import ContentSection from '../../components/shared/ContentSection';
+import LinkButton from '../../components/shared/LinkButton';
+import { below } from '../../utilities/breakpoint';
+
+const GET_PARTNERS = gql`
+  query getPartners {
+    partners {
+      id
+      year
+      partnershipLevel
+      companyName
+      companyLogo
+      heroImage
+      website
+    }
+  }
+`;
+
+const RobotImage = styled.img`
+  height: 500px;
+  float: right;
+  margin-top: -200px;
+  margin-right: 35px;
+
+  ${below.med`
+    margin-top: 20px;
+    margin-right: unset;
+    height: 400px;
+  `};
+`;
+
+const PartnerLevelTitle = styled.h3`
+  margin-top: 0;
+  margin-bottom: 0;
+  text-align: center;
+  text-transform: uppercase;
+`;
+
+const Partners = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+`;
+
+const ImageContainer = styled.div`
+  margin-top: 50px;
+  text-align: center;
+  display: grid;
+  background-color: #fafafa;
+  width: ${({ width }) => width};
+  height: ${({ height }) => height};
+`;
+
+const Image = styled.img`
+  margin: auto;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  max-width: ${({ maxWidth }) => maxWidth};
+`;
+
+const renderPartner = (
+  partner,
+  containerWidth,
+  containerHeight,
+  imageMaxWidth,
+) => {
+  const url = `/wi/partner-detail?id=${partner.id}`;
```

Comment:
@csell5 I know you are thinking about this for the routing, this is where you want to use a slug in the url, right? Ex: `/wi/parters/acme`?